README.md

```diff
@@ -384,19 +384,23 @@ amadeus.booking.hotelBookings.post(
 
 // Points of Interest
 // What are the popular places in Barcelona (based a geo location and a radius)
-amadeus.referenceData.locations.pointsOfInterest.get({
+amadeus.referenceData.locations.pointsOfInterest().get({
     latitude : 41.397158,
     longitude : 2.160873
 })
 
 // What are the popular places in Barcelona? (based on a square)
-amadeus.referenceData.locations.pointsOfInterest.bySquare.get({
+amadeus.referenceData.locations.pointsOfInterest().bySquare.get({
     north: 41.397158,
     west: 2.160873,
     south: 41.394582,
     east: 2.177181
 })
 
+// Points of Interest
+// Extract the information about point of interest with ID '9CB40CB5D0'
+amadeus.referenceData.locations.pointsOfInterest('9CB40CB5D0').get()
```

Comment:
I understood this later in the PR but usually for that case we use a singular for the retreive instead of a plural (for example location and locations) that will solve the problem with all the () as well

Reply:
This is to maintain consistency between sub-URLs and SDK's signatures. If sub-path _/reference-data/locations/pois_ is represented by _.referenceData.locations.pointsOfInterest()_ for one endpoint, then it should do so for all other APIs. Please refer tripParserJobs API - which has similar implementation, post an e-mail discussion on the topic.

Reply:
Now that you mention it it is true that I think TripParserJobs should have had a singular one like locations/location and Hotel Search API

```node
// Confirm the availability of a specific offer id
amadeus.shopping.hotelOffer('XXX').get()
```

I will leave it to Alvaro for final decision as he worked more on the Node SDK